app/bundles/FormBundle/Resources/views/Builder/form.html.twig

```diff
@@ -66,29 +66,15 @@
                       {%- set template = '@MauticForm/Field/' ~ f.type ~ '.html.twig' -%}
                   {%- endif %}
                 
-                {%- if theme is empty -%}                    
-                  <!-- start: "{{ theme }}{{ template }}" -->
-                  {{- include(theme ~ f.type ~ '.html.twig', {
-                          'field': f.convertToArray,
-                          'id': f.alias,
-                          'formName': formName,
-                          'fieldPage': (pageCount - 1),
-                          'mappedFields': mappedFields,
-                          'inBuilder': inBuilder,
-                          'fields': fields,
+                  {{- include([theme ~ f.type ~ '.html.twig', template], {
```

Comment:
This syntax allows Twig to try if the template exists in the theme and if not, use the default one. See https://twig.symfony.com/doc/1.x/tags/include.html#:~:text=You%20can%20also%20provide%20a%20list%20of%20templates%20that%20are%20checked%20for%20existence%20before%20inclusion.%20The%20first%20template%20that%20exists%20will%20be%20included